src/shared/common/components/workstack.jsx

```diff
@@ -312,7 +312,7 @@ class WorkstackAllocate extends Component {
                 }
                 return <td key={row.uuid + column.dataValueKey} className='govuk-table__cell'>{value}</td>;
             case ColumnRenderer.DUE_DATE_WARNING:
-                if (row.data.CaseContributions) {
+                if (row.data.CaseContributions  && row.data.ContributionsRequired !== 'N') {
```

Comment:
Is the `ContributionsRequired` property available for MPAM also? Just thinking it may break MPAM contributions if not.

Reply:
I've tested this and the column still appears correctly on MPAM. If row row.data.ContributionsRequired is missing it will evaluate to undefined, which is still still satisfies the condition of not equalling 'N'.